### PR TITLE
FIX: Mongo wasn't utilizing whitelist/blacklist feature

### PR DESCRIPTION
**Pull request by zthatch**
Mongo was loading all databases in the list every time. Tests pass locally, including mongo helpers.

### CODE REVIEW

File: regolith/builders/activitylogbuilder.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ class ActivitylogBuilder(LatexBuilderBase):
     """Build CV from database entries"""
 
     btype = "annual-activity"
-    needed_dbs = ['groups', 'people', 'grants', 'proposals', 'institutions',
+    needed_cols = ['groups', 'people', 'grants', 'proposals', 'institutions',
```
Comment (sbillinge) [inline regolith/builders/activitylogbuilder.py:35]: I think `colls` (short for `collections`) might be better but not sure.  What do you think?

**Review by sbillinge — COMMENTED** [review]
one comment....I think I prefer `needed_colls` ....

File: regolith/builders/activitylogbuilder.py (hunk 1 above)
Comment (zthatch) [reply regolith/builders/activitylogbuilder.py:35]: can do

**Comment by sbillinge** [comment]
thanks @zthatch

**Pull request merged**